tools/ci.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
As a Nix user, I prefer the more portable `env` version :stuck_out_tongue: 

```suggestion
#!/usr/bin/env bash
```

Reply:
Ok, fair enough, if we change this we should change every instance, and this can be done in a separate PR. My previous comment can be ignored